CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
 PROJECT(tiny_cnn)
-set(PROJECT_VERSION "0.0.1")
+set(PROJECT_VERSION "0.1.0")
 string(REGEX MATCHALL "[0-9]" PROJECT_VERSION_PARTS "${PROJECT_VERSION}")
```

Comment:
No, I rebase this codes still on feat/generic-computational-graph, is the version should be remained as 0.0.1 except for master branch?